tests/test_update.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import pytest
+from sphinx.application import Sphinx
+
+
+SIMPLE_EXPORTED_FNAME = 'drawio-bf0f85b68784bab0e62bf5902f5a46b65d71ee70.png'
+
+
+@pytest.mark.sphinx("html", testroot="simple")
+def test_notchanged(content: Sphinx):
+    exported = content.outdir / '_images' / SIMPLE_EXPORTED_FNAME
```

Comment:
Please try to use single quotes not double quotes.

Reply:
Sorry, I have the habit of using single quotes. Fixed now.